src/ResultView.tsx

```diff
@@ -10,7 +11,115 @@ const ResultView = ({ result }: { result: any }) => {
     role,
     school,
   } = result;
-
+  const MobileView = useMediaQuery("(max-width:800px)");
+  if (MobileView) {
+    return (
+      <div>
+        <li className="sui-result">
+          <div className="sui-result__header">
+            <span
+              className="sui-result__title"
+              dangerouslySetInnerHTML={{
+                __html: name && name.snippet ? name.snippet : "NA",
+              }}
+            />
```

Comment:
Given the duplication of this code for the title (as well as for all the other fields), would it be possible to create separate reusable React components for them?